package.json

```diff
@@ -37,8 +37,7 @@
     "karma-webpack": "^5.0.0",
     "mocha": "^10.0.0",
     "mocha-lcov-reporter": "^1.3.0",
-    "multibase": "^3.1.0",
-    "multicodec": "^3.2.1",
+    "multiformats": "^11.0.1",
```

Comment:
Yes, I considered this as well, but figured I would use `multiformats` only because it was recommended from the deprecated `multicodec`. However, it is true that using `base58` `encode`/`decode` also works and may be the best option considering that we are already using these functions elsewhere in the code. @gannan08 thoughts?

Reply:
We should use `base58-universal` and reduce the number of dependencies.

Reply:
I can live with just using `base58-universal` in this case.